docs/sdk/src/polkadot_sdk/xcm.rs

```diff
@@ -1,5 +1,38 @@
 //! # XCM
 //!
-//! @KiChjang @franciscoaguirre
-//! TODO: RFCs, xcm-spec, the future of the repo, minimal example perhaps, forward to where actual
-//! docs are hosted. https://github.com/paritytech/polkadot-sdk-docs/issues/58
+//! XCM, or Cross-Consensus Messaging, is a **language** to communicate **intentions** between **consensus systems**.
+//!
+//! ## Overview
+//!
+//! XCM is a language angostic standard, whose specification lives in the [xcm format repo](https://github.com/paritytech/xcm-format).
```

Comment:
You just said it's a language and now you say it's language-agnostic.
```suggestion
//! XCM is a language agnostic standard, whose specification lives in the [xcm format repo](https://github.com/paritytech/xcm-format).
```